package io.inventi.eventstore.eventhandler.exception

class EventAlreadyHandledException(message: String) : RuntimeException(message)